src/client-functions/selector-builder/index.js

```diff
@@ -144,6 +146,22 @@ export default class SelectorBuilder extends ClientFunctionBuilder {
         if (snapshot && !this.options.counterMode) {
             this._addBoundArgsSelectorGetter(snapshot, selectorArgs);
             createSnapshotMethods(snapshot);
+
+            if (this.options.customMethods) {
```

Comment:
There is a consistency problem: It's not snapshot methods, it's selector methods